spaceR/SpacerTests/SpaceRTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  spaceRTests.swift
+//  spaceRTests
+//
+//  Created by Kieran Woodrow on 2022/02/25.
+//
+
+import XCTest
+@testable import spaceR
+
+class spaceRTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
```

Comment:
Ideally we don't want to commit comments to main and develop even though these are automatically generated when you create the test files